hummingbot/strategy/pure_market_making/pure_market_making_config_map.py

```diff
@@ -288,28 +296,38 @@ def exchange_on_validated(value: str):
                   default="current_market",
                   validator=validate_price_source,
                   on_validated=on_validate_price_source),
+    "price_type":
+        ConfigVar(key="price_type",
+                  prompt="Which price type to use? (mid_price/last_price) >>> ",
```

Comment:
Options here should be mid_price/last_price/best_bid/best_ask (as many as PriceType enum has).

Reply:
I started making the change, but I think that `AssetPriceDelegate` only offers "mid_price". How should we proceed in that case? External market always = mid-price for now?

Reply:
These new four price types should apply to external market as well. 
Can you please modify the `AssetPriceDelegate`? Thanks